vagrant.default.yml

```diff
@@ -3,7 +3,8 @@ vagrant_ip: '192.168.56.5'
 vagrant_cpus: 1
 vagrant_memory: 1024 # in MB
 vagrant_box: 'bento/ubuntu-20.04'
-vagrant_box_version: '>= 202012.23.0'
+vagrant_box_version: '>= 0'
```

Comment:
Why `>= 0`?

Reply:
I guess it doesn't matter as much now that we're using the main `bento` boxes which all have consistent date based version numbers. But before if you override `vagrant_box` to an arm64 one from a different provider, this version constraint would prevent you from using it so you'd have to overwrite it as well.

`'>= 0` just means "give me the latest box" which honestly isn't too different than what we had since we never actually bump this.